### PR TITLE
CNV 2.3 release notes

### PR DESCRIPTION
**Pull request by aburdenthehand**
Went through the 2.2 known issues and removed all the ones that have been resolved. It looks as though there are still eight bugs that are still relevant for 2.3

### CODE REVIEW

File: cnv/cnv_release_notes/cnv-release-notes.adoc [hunk 1]
```diff
@@ -15,49 +15,19 @@ include::modules/technology-preview.adoc[leveloffset=+2]
```
Comment (ousleyp) [inline cnv/cnv_release_notes/cnv-release-notes.adoc:15]: Line 13: `:FeatureName: {CNVProductName}` needs to be changed to `:FeatureName: {CNVProductNameStart}`

**Review by ousleyp — COMMENTED** [review]
One minor issue with the existing file that I think should be fixed in this PR, but otherwise LGTM.

While it's on my mind, we need to increment {CNVVersion} to 2.3 in the attributes file; want to do that in this PR too?

**Comment by aburdenthehand** [comment]
@ousleyp Thanks for the eyes, Pan. I updated as suggested. 
I might hold off merging this so I can merge #20823 first.

**Comment by aburdenthehand** [comment]
Manual cherrypick to enterprise-4.4 #20920

**Pull request merged**